src/components/TransactionList/TransactionList.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import transactions from "./transactions.json";
+import TransactionItem from "../TransactionItem/TransactionItem";
+
+const TransactionList = () => {
+  console.log(transactions);
+
+  const renderList = transactions.map((transaction) => (
```

Comment:
Create one more component, and then pass those transactions through props, then you would be able to write tests for this component and now you have strong dependency to this json file